powerdnsadmin/__init__.py

```diff
@@ -78,6 +79,11 @@ def create_app(config=None):
         from flask_sslify import SSLify
         _sslify = SSLify(app)  # lgtm [py/unused-local-variable]
 
+    # Load Flask-Session
+    if app.config.get("SESSION_TYPE") is not None and app.config.get("SESSION_TYPE") != 'null':
```

Comment:
Can we just use
```python
app.config.get("SESSION_TYPE")
```
instead of
```python
app.config.get("SESSION_TYPE") is not None
```
And, why it cannot be `null` string? What is the possible input for this `SESSION_TYPE` env variable? Can we validate them?

Reply:
Yep, your suggestion for app.config.get("SESSION_TYPE") makes perfect sense.

The only reason I excluded null was that is appears the flask-session requires a SECRET_KEY if you set the session_type to null (or at least my container kept getting errors about the session_key when the SESSION_TYPE was set to null.  
I had considered including a new variable for SECRET_KEY or using the existing SECRET_KEY variable, but I didn't want to break existing builds by doing this.  Happy to do either though really.

Reply:
I've removed the untested session type and used a FILESYSTEM_SESSIONS_ENABLE configuration instead, there is validation for this being boolean.